package.json

```diff
@@ -64,8 +65,8 @@
     "jest-matcher-utils": "^22.0.0"
   },
   "lint-staged": {
-    "*.js": [
-      "yarn prettier",
+    "src/**/*.js": [
+      "yarn run lint:fix",
       "git add"
```

Comment:
we don't need the `git add` I think

Reply:
I wondered about that. It was present before, but I've never had a setup like that.